composer.json

```diff
@@ -117,14 +122,19 @@
         "test": "@test:all",
         "test:all": [
```

Comment:
Name of this script should be changed because it's no longer running all scripts (suite `tests` excludes auto-review and smoke tests). Maybe it's good idea to extract unit and integration tests to separate suites and introduce `test:unit` and `test:integration`, then `test:all` could be a group that would invoke `test:unit`, `test:integration`, and `test:smoke`? If you consider smoke tests out of the scope of basic test suite, then probably it's fine as-is (but introducing unit/integration suites still relevant).

Reply:
I was struggling here too, it was already not running all tests actually, if you look under the hood.
I wouldn't put smoke tests to be run by default, tbh. they require phar build and composer repo cloning

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7407/commits/004257a766cf9802c97304aace8a15976a2e3d22